lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -171,6 +175,21 @@ protected void ValidateSuccessForLocalRuntime(int numberOfContextsToClose, int n
         }
 
         /// <summary>
+        /// Find the number of log lines that contains the message in the log file specified
+        /// </summary>
+        /// <param name="message"></param>
+        /// <param name="subfolder"></param>
+        /// <param name="fileName"></param>
+        /// <param name="testFolder"></param>
+        /// <returns></returns>
+        protected int GetMessageCount(string message, string subfolder, string fileName, string testFolder)
+        {
+            string[] lines = ReadLogFile(fileName, subfolder, testFolder);
+            string[] linesContainsMessage = lines.Where(s => s.Contains(message)).ToArray();
+            return linesContainsMessage.Length;
```

Comment:
These two lines can be replaced with 

    return lines.Where(s => s.Contains(message)).Count();